.luacheckrc

```diff
@@ -0,0 +1,29 @@
+std = "luajit"
```

Comment:
It works with upstream luacheck but there is tarantool/luacheck fork that supports std="tarantool". (https://github.com/tarantool/luacheck/commit/2a0f1d1094527c80604a36bb47a7760b69be628a)

But seems it doesn't matter. Patch looks good.

```
➜  vshard git:(luacheck) ✗ luacheck .
Checking test/router-luatest/router_test.lua      OK
Checking test/unit-luatest/version_test.lua       OK
Checking vshard/cfg.lua                           OK
Checking vshard/consts.lua                        OK
Checking vshard/error.lua                         OK
Checking vshard/hash.lua                          OK
Checking vshard/heap.lua                          OK
Checking vshard/init.lua                          OK
Checking vshard/registry.lua                      OK
Checking vshard/replicaset.lua                    OK
Checking vshard/rlist.lua                         OK
Checking vshard/router/init.lua                   OK
Checking vshard/storage/init.lua                  OK
Checking vshard/storage/ref.lua                   OK
Checking vshard/storage/reload_evolution.lua      OK
Checking vshard/storage/sched.lua                 OK
Checking vshard/util.lua                          OK
Checking vshard/version.lua                       OK

Total: 0 warnings / 0 errors in 18 files
```

However to completely close problems with uninitialized variables we should add luacheck to CI.